examples/losses/maeloss.py

```diff
@@ -0,0 +1,10 @@
+import torch.nn as nn
+
+class MAELoss(nn.Module):
+    '''Mean Squared Error Loss'''
```

Comment:
I guess the comment here should be Mean Absolute Error Loss?

Reply:
right, my bad